ffi/Makefile

```diff
@@ -0,0 +1,9 @@
+clean:
+	@rm -rf bin go.mod go.sum
+
+GO_SRC=./ffi_wrapper.go
+
+compile:
```

Comment:
What's the plan around this? An example of how to build it under Linux & users can work out from this how to do it on their own OS?

Reply:
2) I would argue the output is not something for `bin` - it's a `lib`